op-conductor/conductor/config.go

```diff
@@ -78,6 +78,12 @@ type Config struct {
 	// RPCEnableProxy is true if the sequencer RPC proxy should be enabled.
 	RPCEnableProxy bool
 
+	// RollupBoostWsURL is the URL of the rollup boost websocket proxy.
+	RollupBoostWsURL string
+
+	// WebsocketServerPort is the port of the websocket server.
```

Comment:
Nit: Can we have more explanatory comment emphasizing that the websocket server is not some other server but something exposed by op-conductor itself?

Such as "WebsocketServerPort is the port at which op-conductor exposes its websocket server from which clients can read streams sourced from rollupBoostWsUrl"